pgml-extension/sql/pgml--2.7.6--2.7.7.sql

```diff
@@ -0,0 +1,8 @@
+-- Drop the constraint on snapshot_id for models
```

Comment:
Gonna need to bump the version in `Cargo.toml` and hit build to update the lockfile.